index.html

```diff
@@ -1,55 +1,237 @@
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+          <div class="box">
+            <div></div>
+            <div class="box-menu">
+              <h1 class="title">Multi-Post Stories Gain+Glory</h1>
+              <ul class="box-list">
+                <li class="box-list-item">Ruby on rails</li>
+                <li class="box-list-item">css</li>
+                <li class="box-list-item">Javascript</li>
+                <li class="box-list-item">html</li>
+              </ul>
+              <button class="box-button">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="third-section">
+      <div class="container">
+        <h1>About Me</h1>
+        <p class="about-desc">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="box-button about-btn">Get My Resume</button>
+      </div>
```

Comment:
- [ ] Dear @danutibe07, Please don't forget to add the decoration images here. I cannot see the images now. And please don't forget all decoration images should be a background image. 

### Figma design 
![Screenshot 2023-01-13 093747](https://user-images.githubusercontent.com/69077061/212254267-86c6cab1-fbb2-4cf1-8361-4dc4e9b49069.png)

### Your design
![Screenshot 2023-01-13 093808](https://user-images.githubusercontent.com/69077061/212254270-04588e01-294f-4316-bb9a-4796223704eb.png)